cluster/gce/gci/configure-helper.sh

```diff
@@ -409,6 +409,23 @@ function assemble-docker-flags {
   fi
 
   echo "DOCKER_OPTS=\"${docker_opts} ${EXTRA_DOCKER_OPTS:-}\"" > /etc/default/docker
+
+  # If using a network plugin, extend the docker configuration to always remove
+  # the network checkpoint to avoid corrupt checkpoints.
+  # (https://github.com/docker/docker/issues/18283).
+  if [[ "${use_net_plugin}" == "true" ]]; then
```

Comment:
If user wants to use docker networking (without any plugins), we can't freely nuke the checkpoint every time docker restarts. We'll need to be able to detect the corrupt checkpoint.

The restart is done immediately below, so I didn't do it again here.